app/Http/Controllers/CustomizerController.php

```diff
@@ -210,13 +210,6 @@ protected function prepSeed(Request $request, bool $save = false)
                 $locations[$decoded_location]->setItem($place_item);
             }
         }
-        // force starting bomb and quiver capacities
-        $world->setPreCollectedItems(new ItemCollection([
-            Item::get('BombUpgrade10', $world),
-            Item::get('ArrowUpgrade10', $world),
-            Item::get('ArrowUpgrade10', $world),
-            Item::get('ArrowUpgrade10', $world),
-        ]));
```

Comment:
Are we relying on a different mech? (how does this work with customizer?)

Reply:
Basically there was a conflict with this and the ammo upgrade counters in SRAM which is why I changed it. So the starting max also gets set separately in the Rom.php (StartingMaxArrows and StartingMaxBombs in tables.asm.) Then it will work as normal with no starting upgrades but they can be added as starting items and will be applied correctly in customizer as well. The world model and filler don't depend on these being pre-collected as far as I can tell.